### ISSUE TITLE
Fragrances

### ISSUE THREAD

**Issue by nicolasfoss** [rfordatascience/tidytuesday#774]
Hey Jon!

I saw a post on LinkedIn suggesting this dataset. 

Has TT ever looked at fragrances?

https://www.kaggle.com/datasets/olgagmiufana1/parfumo-fragrance-dataset

https://github.com/rdemarqui/perfume_recommender/tree/main

I have cloned the repo and would be glad to submit a pull request. 

~ Dr. Nic

**Comment by jonthegeek** [comment]
Not as far as I can see:

``` r
ttmeta::tt_summary_tbl |> 
  dplyr::filter(
    stringr::str_detect(tolower(title), "fragrance") |
    stringr::str_detect(tolower(title), "perfume")
  )
#> # A tibble: 0 × 8
#> # ℹ 8 variables: year <int>, week <int>, date <date>, title <chr>,
#> #   source_title <chr>, source_urls <list>, article_title <chr>,
#> #   article_urls <list>

ttmeta::tt_urls_tbl |> 
  dplyr::filter(
    domain == "kaggle",
    stringr::str_detect(path, "fragrance")
  )
#> # A tibble: 0 × 11
#> # ℹ 11 variables: year <int>, week <int>, type <fct>, url <chr>, scheme <fct>,
#> #   domain <chr>, subdomain <chr>, tld <chr>, path <chr>, query <list>,
#> #   fragment <chr>

ttmeta::tt_urls_tbl |> 
  dplyr::filter(
    domain == "github",
    stringr::str_detect(path, "perfume")
  )
#> # A tibble: 0 × 11
#> # ℹ 11 variables: year <int>, week <int>, type <fct>, url <chr>, scheme <fct>,
#> #   domain <chr>, subdomain <chr>, tld <chr>, path <chr>, query <list>,
#> #   fragment <chr>
```

<sup>Created on 2024-11-24 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

We'd love a PR!

**Comment by nicolasfoss** [comment]
@jonthegeek ask, and you will receive!

Thank you for your help.

**Comment by jonthegeek** [comment]
Gonna leave this open to connect it up once you submit the PR. Thanks!

**Comment by nicolasfoss** [comment]
Sounds good, Jon. Thank you.

**Comment by nicolasfoss** [comment]
Should we close this one, now?

**Comment by jonthegeek** [comment]
Closed via #776
